packages/itwinui-react/jest.config.cjs

```diff
@@ -150,7 +150,13 @@ module.exports = {
   // ],
 
   // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
-  testPathIgnorePatterns: ['/node_modules/', '/esm/', '/cjs/'],
+  testPathIgnorePatterns: [
+    '/node_modules/',
+    '/esm/',
+    '/cjs/',
+    // Since this file is only to test types and not to run any unit tests
+    'src/core/Table/types/react-table-types.test.tsx',
```

Comment:
nit: this config change could be avoided if you use a different filename convention

for example `react-table.types-test.tsx`